### PR TITLE
messing around

### PR DESCRIPTION
**Pull request by mitchbregs**
You already know

### CODE REVIEW

**Comment by mitchbregs** [comment]
Screen Shot 2020-02-15 at 3 00 56 PM

**Review by leonkozlowski — APPROVED** [review]
L G T M

File: src/jiggy/runner.py [hunk 1]
```diff
@@ -113,4 +113,22 @@ def _execute(self, pkg: str, mdl: str, node: Node, inputs=None):
 
     def run(self):
         """Abstract runner for Sequential."""
+
+        logo = """
```
Comment (leonkozlowski) [inline src/jiggy/runner.py:117]: Love this

**Review by leonkozlowski — APPROVED** [review]
Re: LGTM

**Pull request merged**